### ISSUE TITLE
windows 10 rs 2 when you click Start nvda occasionally hangs and helps her reboot.

### ISSUE THREAD

**Issue by gdata1** [nvaccess/nvda#7056]
hello windows 10 rs2 When you click on the windows button sometimes freezes nvda helps only its restart nvda. the same problem sometimes happens in Microsoft edge
windows 10 rs2 15063.14.
nvda next-13972,c956204f

**Comment by PratikP1** [comment]
I am unable to replicate with Windows 10 Professional. Is it possible to get a Log?

**Comment by josephsl** [comment]
Hi, do you have a log so we can take a look? Thanks.

**Comment by gdata1** [comment]
Unfortunately I can't provide the logs. When this problem happens I rerun the nvda and logs updated.

**Comment by PratikP1** [comment]
You should be able to get NVDA’s latest log after restart in your C:\users\user name\local\temp directory. Check the date and time on that log.

**Comment by gdata1** [comment]
ERROR - eventHandler.executeEvent (18:08:33):
error executing event: gainFocus on <NVDAObjects.IAccessible.ContentGenericClient object at 0x04FCD950> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 138, in executeEvent
  File "eventHandler.pyc", line 153, in doPreGainFocus
  File "NVDAObjects\__init__.pyc", line 278, in objectInForeground
  File "NVDAObjects\__init__.pyc", line 190, in findBestAPIClass
  File "NVDAObjects\__init__.pyc", line 189, in findBestAPIClass
  File "NVDAObjects\UIA\__init__.pyc", line 766, in kwargsFromSuper
COMError: (-2146233083, None, (None, None, None, 0, None))
ERROR - eventHandler.executeEvent (18:55:56):
error executing event: typedCharacter on <NVDAObjects.Dynamic_SearchFieldEditableTextWithoutAutoSelectDetectionUIA object at 0x050440B0> with extra args of {'ch': u'\x17'}
Traceback (most recent call last):
  File "eventHandler.pyc", line 143, in executeEvent
  File "eventHandler.pyc", line 91, in __init__
  File "eventHandler.pyc", line 98, in next
  File "NVDAObjects\__init__.pyc", line 847, in event_typedCharacter
  File "speech.pyc", line 625, in speakTypedCharacters
  File "api.pyc", line 239, in isTypingProtected
  File "baseObject.pyc", line 34, in __get__
  File "baseObject.pyc", line 110, in _getPropertyViaCache
  File "NVDAObjects\UIA\__init__.pyc", line 958, in _get_states
  File "NVDAObjects\UIA\__init__.pyc", line 680, in _prefetchUIACacheForPropertyIDs
COMError: (-2147467259, '\xcd\xe5\xee\xef\xee\xe7\xed\xe0\xed\xed\xe0\xff \xee\xf8\xe8\xe1\xea\xe0', (None, None, None, 0, None))
INFO - core.main (18:56:01):
Exiting
INFO - config.ConfigManager.save (18:56:01):
Base configuration saved
INFO - __main__ (18:56:04):
NVDA exit

**Comment by gdata1** [comment]
and here's another recent errors.
ERROR - eventHandler.executeEvent (19:35:18):
error executing event: gainFocus on <NVDAObjects.Dynamic_EditableTextWithoutAutoSelectDetectionEdgeHTMLRootUIA object at 0x04FD3BD0> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 138, in executeEvent
  File "eventHandler.pyc", line 171, in doPreGainFocus
  File "browseMode.pyc", line 1070, in event_treeInterceptor_gainFocus
  File "browseMode.pyc", line 1126, in _set_selection
  File "cursorManager.pyc", line 107, in _set_selection
  File "cursorManager.pyc", line 410, in updateSelection
  File "treeInterceptorHandler.pyc", line 158, in copy
  File "NVDAObjects\UIA\__init__.pyc", line 608, in copy
  File "NVDAObjects\UIA\__init__.pyc", line 235, in __init__
  File "textInfos\__init__.pyc", line 238, in __init__
TypeError: cannot create weak reference to 'NoneType' object
ERROR - eventHandler.executeEvent (21:32:29):
error executing event: typedCharacter on <NVDAObjects.UIA.UIA object at 0x051E99B0> with extra args of {'ch': u'\t'}
Traceback (most recent call last):
  File "eventHandler.pyc", line 143, in executeEvent
  File "eventHandler.pyc", line 91, in __init__
  File "eventHandler.pyc", line 98, in next
  File "NVDAObjects\__init__.pyc", line 847, in event_typedCharacter
  File "speech.pyc", line 625, in speakTypedCharacters
  File "api.pyc", line 239, in isTypingProtected
  File "baseObject.pyc", line 34, in __get__
  File "baseObject.pyc", line 110, in _getPropertyViaCache
  File "NVDAObjects\UIA\__init__.pyc", line 958, in _get_states
  File "NVDAObjects\UIA\__init__.pyc", line 680, in _prefetchUIACacheForPropertyIDs
COMError: (-2146233083, None, (None, None, None, 0, None))

**Comment by gdata1** [comment]
Here's the log at the time of the problem.
WARNING - watchdog._watcher (23:50:56):
Core frozen in stack:
  File "nvda.pyw", line 195, in <module>
  File "core.pyc", line 396, in main
  File "wx\_core.pyc", line 8657, in MainLoop
  File "wx\_core.pyc", line 7952, in MainLoop
  File "core.pyc", line 366, in Notify
  File "IAccessibleHandler.pyc", line 888, in pumpAll
  File "IAccessibleHandler.pyc", line 755, in processForegroundWinEvent
  File "IAccessibleHandler.pyc", line 522, in winEventToNVDAEvent
  File "_UIAHandler.pyc", line 299, in isUIAWindow
  File "_UIAHandler.pyc", line 293, in _isUIAWindowHelper

WARNING - watchdog._watcher (23:51:11):
Core frozen in stack:
  File "nvda.pyw", line 195, in <module>
  File "core.pyc", line 396, in main
  File "wx\_core.pyc", line 8657, in MainLoop
  File "wx\_core.pyc", line 7952, in MainLoop
  File "core.pyc", line 366, in Notify
  File "IAccessibleHandler.pyc", line 888, in pumpAll
  File "IAccessibleHandler.pyc", line 755, in processForegroundWinEvent
  File "IAccessibleHandler.pyc", line 522, in winEventToNVDAEvent
  File "_UIAHandler.pyc", line 299, in isUIAWindow
  File "_UIAHandler.pyc", line 293, in _isUIAWindowHelper

**Comment by gdata1** [comment]
Hello. appears in someone my problem? the log I posted.

**Comment by ehollig** [comment]
@gdata1, is this still an issue? @josephsl and @PratikP1, do the logs provided above help in any way in tracking down the cause of this issue?

**Comment by Adriani90** [comment]
@gdata1 there were several issues with regard to this problem and they have been closed because people, including myself, are not having the problem anymore. are you still having this issue?

**Comment by gdata1** [comment]
No.
